src/Umbraco.Web.UI.Client/src/common/directives/components/content/edit.controller.js

```diff
@@ -815,7 +815,7 @@
 
                     //ensure the save flag is set
                     selectedVariant.save = true;
-                    performSave({ saveMethod: contentResource.publish, action: "save" }).then(function (data) {
+                    performSave({ saveMethod: $scope.saveMethod(), action: "save" }).then(function (data) {
```

Comment:
@kjac is this right? don't we want `$scope.saveMethod` not `$scope.saveMethod()` ? We want to pass the function, not execute it

Reply:
Hi @Shazwazza ... I know it looks wonky but `$scope.saveMethod()` is correct. It's also how it's done throughout the controller - for example in the [`save`](https://github.com/umbraco/Umbraco-CMS/blob/temp8/src/Umbraco.Web.UI.Client/src/common/directives/components/content/edit.controller.js#L618) method.